installation_tips/README.md

```diff
@@ -1,36 +1,53 @@
 ## Installation tips
 
-If you are not (yet) an expert in Python installations (conda vs pip, mananging environements, etc.),
-here we propose a simple recipe to install `spikeinterface` and several sorters inside an anaconda
-environment for windows/mac users.
+If you are not (yet) an expert in Python installations, a main difficulty is choosing the installation procedure.
+The main ideas you need to know before starting:
+ * python itself can be distributed and installed many many ways.
+ * python itself do not contain so many features for scientifique computing you need to install "packages".
+   Numpy, scipy, matplotlib, spikeinterface, ... are python packages that have a complicated dependency graph between then. "uv"
+ * installing package  can be distributed and installed several ways (pip, conda, uv, mamba, ...)
+ * installing many packages at once is challenging (because of the depenency graph) so you need to do it in an "isolated environement"
```

Comment:
```suggestion
 * installing many packages at once is challenging (because of their dependency graphs) so you need to do it in an "isolated environement"
```